exercises/java-primitives/src/main/java/com/cbfacademy/ArithmeticExercise.java

```diff
@@ -0,0 +1,33 @@
+package com.cbfacademy;
+
+public class ArithmeticExercise {
+    
+        float AddFloat(Float operand1, Float operand2) {
+            float add = operand1 + operand2;
+            return add;
+
+        }
```

Comment:
This could be written as: 

```
float AddFloat(Float operand1, Float operand2) {
            return operand1 + operand2;
        }
```

How could you update the remaining methods in the same way?